src/Select/SelectItem.svelte

```diff
@@ -14,6 +14,14 @@
   /** Set to `true` to disable the option */
   export let disabled = false;
 
+  let className = "";
+
+  /** Specify the class of the `option` element */
+  export { className as class };
+
+  /** Specify the style of the `option` element */
+  export let style = "";
```

Comment:
Would suggest `undefined` so the attribute is not added unnecessarily.
(Could also be done for `class`, but there already is a Carbon class on the element.

```suggestion
  /**
   * Specify the style of the `option` element
   * @type {string}
   */
  export let style = undefined;
```

Reply:
Good catch – amended.